app/views/kaminari/_last_page.html.erb

```diff
@@ -0,0 +1,3 @@
+<li>
+  <%= link_to_unless current_page.last?, raw(t 'views.pagination.last'), url, {:remote => remote} %>
```

Comment:
why `raw`?  Please use ruby 1.9 style hashes.  Please add spaces inside the braces.